### ISSUE TITLE
Error displaying a StaticArray

### ISSUE THREAD

**Issue by cdsousa** [julia-vscode/julia-vscode#1338]
I'm getting this error in the vscode julia repl:
```
julia> using StaticArrays

julia> SA[1,2]
2-element SArray{TupleERROR: {MethodError: no method matching size(::SArray{Tuple{2},Int64,1,2})
The applicable method may be too new: running in world age 27132, while current world is 27133.
Closest candidates are:
  size(::Union{LinearAlgebra.Adjoint{T,#s15} where #s15<:Union{StaticArray{Tuple{#s12},T,1} where #s12, StaticArray{Tuple{#s13,#s14},T,2} where #s14 where #s13}, LinearAlgebra.Diagonal{T,#s22} where #s22<:(StaticArray{Tuple{#s23},T,1} where #s23), LinearAlgebra.Hermitian{T,#s19} where #s19<:(StaticArray{Tuple{#s20,#s21},T,2} where #s21 where #s20), LinearAlgebra.LowerTriangular{T,#s27} where #s27<:(StaticArray{Tuple{#s28,#s29},T,2} where #s29 where #s28), LinearAlgebra.Symmetric{T,#s16} where #s16<:(StaticArray{Tuple{#s17,#s18},T,2} where #s18 where #s17), LinearAlgebra.Transpose{T,#s12} where #s12<:Union{StaticArray{Tuple{#s12},T,1} where #s12, StaticArray{Tuple{#s13,#s14},T,2} where #s14 where #s13}, LinearAlgebra.UnitLowerTriangular{T,#s33} where #s33<:(StaticArray{Tuple{#s34,#2s35},T,2} where #s35 where #s34), LinearAlgebra.UnitUpperTriangular{T,#s30} where #s30<:(StaticArray{Tuple{#s31,#s32},T,2} where #s32 where #s31), LinearAlgebra.UpperTriangular{T,#s24} where #s24<:(StaticArray{Tuple{#s25,#s26},T,2} where #s26 where #s25), StaticArray{Tuple{#s35},T,1} where #s35, StaticArray{Tuple{#s14,#s13},T,2} where #s13 where #s14, StaticArray{#s35,T,N} where N where #s35<:Tuple} where T) at /Users/csousa/.julia/packages/StaticArrays/mlIi1/src/abstractarray.jl:9 (method too new to be called from this world context.)
  size(::AbstractArray{T,N}, ::Any) where {T, N} at abstractarray.jl:38
  size(::BitArray{1}) at bitarray.jl:99
  ...}
Stacktrace:,
Int64 [1] ,treerender1(,::2SArray}{ with indices TupleSOneTo(2){:2
} ,Int641,1,
2 })2 at /Users/csousa/.vscode-insiders/extensions/julialang.language-julia-insider-0.16.5/scripts/packages/VSCodeServer/src/trees.jl:121

 [2] 
julia>  at /Users/csousa/.vscode-insiders/extensions/julialang.language-julia-insider-0.16.5/scripts/packages/VSCodeServer/src/VSCodeServer.jl:56
 [3] handle_message(; crashreporting_pipename::String) at /Users/csousa/.vscode-insiders/extensions/julialang.language-julia-insider-0.16.5/scripts/packages/VSCodeServer/src/VSCodeServer.jl:285
 [4] macro expansion at /Users/csousa/.vscode-insiders/extensions/julialang.language-julia-insider-0.16.5/scripts/packages/VSCodeServer/src/VSCodeServer.jl:207 [inlined]
 [5] (::VSCodeServer.var"#40#42"{String})() at ./task.jl:358
julia> 
```

- Julia Insider 0.16.5
- Julia Version 1.4.0
- [90137ffa] StaticArrays v0.12.3

**Comment by cdsousa** [comment]
After this happens, it won't execute any code within files (only within REPL).

**Comment by aviatesk** [comment]
yeah, we may want to keep `handle_message` living even after an internal error like this occurs (we actually do this for development mode: https://github.com/julia-vscode/julia-vscode/blob/master/scripts/packages/VSCodeServer/src/VSCodeServer.jl#L196-L212).

**Comment by davidanthoff** [comment]
I think we should make sure we have try catch blocks around every piece of code where we call out to user code (so that we don't crash when there is a bug in some code that we didn't write). But for all of our code, we should crash if there is a bug and send a crash report, so that we can fix it. That at least has been the philosophy for us so far.

**Comment by cdsousa** [comment]
But the actual source of this specific error was something about vscode extension that is/will be investigated, or is it something with StaticArrays?

**Comment by cdsousa** [comment]
That error is now happening to me very frequently so I have restart Julia REPL dozens of times per hour. Is this something I can try to fix?

**Comment by davidanthoff** [comment]
This is a bug in the extension, we should not crash, even if there is a bug in StaticArrays.

**Comment by pfitzseb** [comment]
FWIW, the original bug seems to be fixed on master.

**Comment by aviatesk** [comment]
I guess without #1344 this error would happen for non-dev mode.

**Comment by pfitzseb** [comment]
Ah, makes sense. Didn't realize we were branching on that.

**Comment by aviatesk** [comment]
I introduced the branching within #1319: ref https://github.com/julia-vscode/julia-vscode/pull/1319#discussion_r437816349

**Comment by cdsousa** [comment]
Hi, this has been fixed already, but in 0.16.8 the error reappeared.
Unlike before, it now still lets one execute inline code, but the REPL is spammed with the error making it almost useless.

**Comment by pfitzseb** [comment]
Fixed now.
